.github/workflows/release-please.yml

```diff
@@ -24,6 +24,11 @@ jobs:
           release-type: simple
           package-name: hetzner.hcloud
 
+          # Skip Azure pipeline for release-please commits
+          pull-request-title-pattern: |-
+            chore${scope}: release${component} ${version}
+            [skip azp]
+
```

Comment:
I'm feeling lucky today, let's try this.

Reply:
Is the newline on purpose?

Reply:
Yes, I prefer to have this in the body of the commit.